src/tests/suite/daos_test.c

```diff
@@ -30,8 +30,8 @@
 #include "daos_test.h"
 
 /** All tests in default order (tests that kill nodes must be last) */
-static const char *all_tests = "mpceXViADKCoROdr";
-static const char *all_tests_defined = "mpceXVixADKCoROdr";
+static const char *all_tests = "mpceXVizADKCoROdr";
+static const char *all_tests_defined = "mpceXVizxADKCoROdr";
```

Comment:
I'd consider breaking this up with a macro

/* SET* for lack of better name */
#define SET1 "mpceXVizADKCoROdr"
#define SET2 "x"

static const char *all_tests = SET1;
static const char *all_tests_defined = SET1 SET2;

Reply:
I'm not actually sure why there are two different strings. I'll look into making just one or using your macro approach.